### DISCUSSION TITLE
Attempting to merge a pull request and got 'Something went wrong while executing your query. Please include `D556:7F94:3C32F91:7B083AB:634F139E` when reporting this issue

### DISCUSSION THREAD

**Discussion by ricardoamador** [community/community#36729] [Apps, API and Webhooks]
We  use a bot in order to merge pull requests and ran into the following error when attempting to merge the pull request.

Something went wrong while executing your query. Please include `D556:7F94:3C32F91:7B083AB:634F139E` when reporting this issue

**Comment by ossanna16** [comment]
Hey there @ricardoamador and thank you for asking a good question! I'm glad you found our community 🙂
